ui/src/shared/components/ThresholdMarkers.tsx

```diff
@@ -62,40 +50,41 @@ const ThresholdMarkers: FunctionComponent<Props> = ({
         const onChangeMaxPos = ({y}) => handleDrag(index, 'maxValue', y)
         const onChangeMinPos = ({y}) => handleDrag(index, 'minValue', y)
 
-        switch (threshold.type) {
-          case 'greater':
-            return (
-              <GreaterThresholdMarker
-                key={index}
-                yScale={yScale}
-                yDomain={yDomain}
-                threshold={threshold}
-                onChangePos={onChangePos}
-              />
-            )
-          case 'lesser':
-            return (
-              <LessThresholdMarker
-                key={index}
-                yScale={yScale}
-                yDomain={yDomain}
-                threshold={threshold}
-                onChangePos={onChangePos}
-              />
-            )
-          case 'range':
-            return (
-              <RangeThresholdMarkers
-                key={index}
-                yScale={yScale}
-                yDomain={yDomain}
-                threshold={threshold}
-                onChangeMinPos={onChangeMinPos}
-                onChangeMaxPos={onChangeMaxPos}
-              />
-            )
-          default:
-            return null
+        if (threshold.lowerBound && threshold.upperBound) {
```

Comment:
I imagine we are going to perform this logic in a bunch of places. What do you think about extracting out a helper like

```
const getThresholdType = (threshold: Threshold): 'range' | 'greater' | 'lesser' => {
  // ...
```

Reply:
https://github.com/influxdata/influxdb/issues/14537